packages/react-vapor/src/components/table-hoc/examples/TableHOCExamples.tsx

```diff
@@ -74,7 +74,7 @@ const TableWithFilterAndPagination = _.compose(
     tableWithBlankSlate({title: 'No data caused the table to be empty'}),
     tableWithFilter(),
     tableWithBlankSlate({title: 'Filter caused the table to be empty'}),
-    tableWithPagination({perPageNumbers: [3, 5, 10]})
+    tableWithPagination()

```

Comment:
Here I removed the perPageNumbers specified in the config in this example to show a case where it falls back to the default perPageNumbers.